qiita_pet/webserver.py

```diff
@@ -106,6 +104,7 @@ def __init__(self):
             (r"/study/sharing/", ShareStudyAJAX),
             # ORDER FOR /study/description/ SUBPAGES HERE MATTERS.
             # Same reasoning as below. /study/description/(.*) should be last.
+            (r"/study/description/sample_template/", SampleTemplateAJAX),
```

Comment:
Sorry, what I was expecting to see where handler entries for the API but
they do not appear to exist

On Tue, Dec 22, 2015 at 9:35 AM, Joshua Shorenstein <
notifications@github.com> wrote:

> In qiita_pet/webserver.py
> <https://github.com/biocore/qiita/pull/1582#discussion_r48277143>:
>
> > @@ -106,6 +104,7 @@ def __init__(self):
> >              (r"/study/sharing/", ShareStudyAJAX),
> >              # ORDER FOR /study/description/ SUBPAGES HERE MATTERS.
> >              # Same reasoning as below. /study/description/(.*) should be last.
> > +            (r"/study/description/sample_template/", SampleTemplateAJAX),
>
> These are not the RESTful API endpoints but qiita_pet AJAX call endpoints.
> These allow lazy loading the needed elements on the study info page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1582/files#r48277143>.
>